### PR TITLE
Modified it as a function for future purposes

### PR DESCRIPTION
**Pull request by Shrivaibavi**
Signed-off-by: Shrivaibavi Raghaventhiran <sraghave@redhat.com>

### CODE REVIEW

File: tests/test_pod_exec.py [hunk 1]
```diff
@@ -27,3 +42,8 @@ def main():
     if err:
         print(err)
     print(ret)
+
+def run(**kwargs):
+
+  output = run_command_on_pod(cmd='ceph osd df')
```
Comment (RazTamir) [inline tests/test_pod_exec.py]: I think you want to check the returned value (output) and return 0 based on the return code

File: tests/test_pod_exec.py [hunk 2]
```diff
@@ -1,24 +1,39 @@
+"""
```
Comment (RazTamir) [inline tests/test_pod_exec.py]: Great patch

**Comment by RazTamir** [comment]
@Shrivaibavi Just noticed the PR header - "Modified it as a function for future purposes".
In case you want this as a function for future perposes, it needs to move outside of the test python file (test_pod_exec.py)

File: tests/test_pod_exec.py [hunk 3]
```diff
@@ -1,24 +1,39 @@
...
+    by default the command runs on tool-box pod
+
+    Args:
+        cmd (str): The command to execute on the pod
+        label_selector (str): The label of the pod
+                        By default the label will be "app=rook-ceph-tools"
+
+    Returns:
+        ret (int): The return value
+        out (str): If output exists
```
Comment (ebenahar) [inline tests/test_pod_exec.py]: The function returns an object called 'V1PodList':

kubernetes.client.models.v1_pod_list.V1PodList

**Comment by vasukulkarni** [comment]
Functionality has changed a bit with recent POD class changes and anyways it has merge conflict, Please check the new POD class and reopen a new PR if that doesn't work. Thanks

**Pull request closed without merging**